test/test-util.js

```diff
@@ -90,49 +91,49 @@ test('getLineAndColumnMessage', function(t) {
 });
 
 test('getLineAndColumnMessage with ranges', function(t) {
```

Comment:
We should rename this test now, since it's not testing `getLineAndColumnMessage` anymore :-)